src/GraphQL/Language/AST/Variables.cs

```diff
@@ -27,12 +27,14 @@ public void Add(Variable variable)
         }
 
         /// <summary>
-        /// Returns the first variable with a matching name, or <see langword="null"/> if none are found.
+        /// Returns the first variable with a matching name, or <paramref name="defaultValue"/> if none are found.
         /// </summary>
-        public object ValueFor(string name)
+        public object ValueFor(string name, object defaultValue = null)
         {
             var variable = _variables?.FirstOrDefault(v => v.Name == name);
```

Comment:
I'm not against LINQ, but I've already had a strong reaction to such comparisons during my experiments with performance. Are we going to get rid of such calls?

Reply:
I would hope so. Now I’m just trying to fix a bug in 3.3.